src/textbooks/textbook-form/TextbookForm.scss

```
@@ -48,8 +48,20 @@
   }
 }
 
-.modal-preview-icon {
-  height: 6.25rem;
-  width: 6.25rem;
+.modal-preview {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: $spacer;
+
+  .modal-preview-icon {
+    height: 6.25rem;
+    width: 6.25rem;
+  }
+
+  .modal-preview-text {
+    font-size: 14px;
```

[nit] It's better to use flexible units of measurement
```suggestion
    font-size: .875rem;
```